File: 1_forloop_add_test/forloop_add_test.jl
#!/usr/bin/env julia

function main(toval)
    local sumval = 0
    for i = 0:toval-1
        for j = 0:toval-1
            sumval = sumval + i+(j^1)
        end
    end
    println("Julia: ", sumval);
end

main( parse(Int, ARGS[1]) )

File: 1_forloop_add_test/forloop_add_test.lua
#!/usr/bin/env luajit

function main(toval)
    local sumval = 0
    for i = 0, toval-1 do
        for j = 0, toval-1 do
            sumval = sumval + i+(j^1)
        end
    end
    print("Luajit:", sumval);
end

main( tonumber(arg[1]) )
